plugin.js

```diff
@@ -149,11 +150,11 @@ function processWorkletFunction(t, fun) {
   // if we don't clone other modules won't process parts of newFun defined below
   // this is weird but couldn't find a better way to force transform helper to
   // process the function
-  const workletID = Math.random() * 1e18;
   const clone = t.cloneNode(fun.node);
   const funExpression = t.functionExpression(null, clone.params, clone.body);
 
   const funString = buildWorkletString(t, fun, variables);
+  const workletID = hash(funString);
```

Comment:
can we also now rename the attribute "workletHash" to indicate that this is not an identifier?

Reply:
Done